### PR TITLE
Add acp mod date

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko, this is ready for review.

File: aubreylib/resource.py [hunk 1]
```diff
@@ -242,6 +244,14 @@ def get_metadata_file(self, parsed_mets):
             raise ResourceObjectException("Could not retrieve the " +
                                           "descriptive metadata file.")
 
+    def get_acp_last_modification_date(self, parsed_mets):
+        """Set the acp_modification_date if we get it, or None otherwise."""
+        metsHdr = parsed_mets.getroot().xpath('metsHdr')
+        if len(metsHdr) > 0:
```
Comment (ldko) [inline aubreylib/resource.py]: Is there a reason to use `len` here over `if metsHdr:` ?

File: tests/test_resource.py [hunk 2]
```diff
@@ -229,3 +233,39 @@ def testResourceObjectEmptyResourceType(self, mocked_get_desc_metadata, mocked_f
...
+            b'''<?xml version="1.0" encoding="UTF-8"?><mets><metsHdr
+            LASTMODDATE="2021-09-24T12:45:34Z"></metsHdr></mets>'''
+        )))
+        assert ro.acp_modification_date == '2021-09-24T12:45:34Z'
+        # Should return None for a tree that has metsHdr but no LASTMODDATE
+        ro.get_acp_last_modification_date(etree.parse(BytesIO(
+            # No LASTMODDATE
+            b'<?xml version="1.0" encoding="UTF-8"?><mets><metsHdr></metsHdr></mets>'
+        )))
+        assert ro.acp_modification_date is None
```
Comment (ldko) [inline tests/test_resource.py]: I'm not sure about the necessity of the last two assertions, but it doesn't hurt to have them.

**Review by ldko — COMMENTED** [review]
Tests pass and looks ok. I have a couple of comments anyway. :)

File: aubreylib/resource.py (hunk 1 above)
Comment (somexpert) [reply aubreylib/resource.py]: Nope, not really. I was just bringing the logic over from other tests, but I'll change it :smile:

Comment (somexpert) [reply aubreylib/resource.py]: heh, was mistaken, not from other "tests", but other parts of this code.

**Review by ldko — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**